lib/assessment.rb

```diff
@@ -5,7 +5,12 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.reduce do |item, memo|
```

Comment:
These are in the wrong order. According to the documentation for `reduce`, its signature is `reduce { |memo, obj| block } → obj`